ProjectLighthouse.Servers.Website/Pages/Moderation/NewCasePage.cshtml

```diff
@@ -17,21 +21,62 @@
+        </div>
+        <div class="field">
+            <label style="text-align: left" for="mod-notes">Mod Notes</label>
+            <textarea name="modNotes" id="mod-notes" spellcheck="true" rows="1"></textarea>
+        </div>
+        <div class="field">
+            <label style="text-align: left" for="expires">Expiration</label>
+            <input type="datetime-local" name="expires" id="expires" required />
+        </div>
+        <button type="button" class="ui yellow button" onclick="setPermanent()">Permanent Ban</button>
+    </div>
+    
+    <div class="ui blue segment">
+        <div class="field">
+            <label style="text-align: left" for="mod-history">Moderation History</label>
+            <select class="ui fluid dropdown" id="mod-history">
+                @if (Model.AffectedHistory.Count != 0)
+                {
+                    <option disabled selected>--- Previous moderation history for user @Model.AffectedUser.Username (@Model.AffectedId) ---</option>
+                    @foreach (ModerationCaseEntity moderationCase in Model.AffectedHistory)
+                    {
+                        <option disabled style="color: black;">
+                            @moderationCase.Type.ToString() by @moderationCase.CreatorUsername
+                            on @TimeZoneInfo.ConvertTime(moderationCase.CreatedAt, timeZoneInfo).ToString("M/d/yyyy @ h:mm tt")
+                            with reason "@(!string.IsNullOrWhiteSpace(moderationCase.Reason) ? moderationCase.Reason : "No reason provided")"
+                        </option>
+                    }
+                }
+                else
+                {
+                    <option disabled selected>--- No moderation history found for user @Model.AffectedUser.Username (@Model.AffectedId) ---</option>
+                }
+            </select>
+        </div>
+    </div>
 
-    <div class="ui left labeled input">
-        <label for="reason" class="ui blue label">Reason: </label>
-        <input type="text" name="reason" id="reason"/>
-    </div><br/><br/>
-
-    <div class="ui left labeled input">
-        <label for="mod-notes" class="ui blue label">Mod Notes: </label>
-        <input type="text" name="modNotes" id="mod-notes"/>
-    </div><br/><br/>
+    <div class="ui red segment">
+        <div style="margin-bottom: 1em;">
+            <i class="ui icon warning"></i><i>Remember to dismiss your case after expiration otherwise it will remain in effect!</i>
+        </div>
+        <button type="submit" class="ui red button">Create Case</button>
+    </div>
+</form> 
 
-    <div class="ui left labeled input">
-        <label for="expires" class="ui blue label">Expires: </label>
-        <input type="datetime-local" name="expires" id="expires"/>
-    </div><br/><br/>
-
-    <br/><input type="submit" value="Create case" class="ui red button"/>
-</form>
\ No newline at end of file
+<script>
+    function setPermanent() {
```

Comment:
Nice jank 😃

Reply:
This isn't really criticism, though if it's permanent surely a cleaner solution would be a boolean?

Reply:
Elaborate?

Reply:
Never mind I know what to do